list_to_email_converter.py

```diff
@@ -0,0 +1,215 @@
+import os
+from urllib.parse import quote
+import sys
+import argparse
+
+user_input = input("Write the path of your file or folder: ")
+
+if user_input.find('.md') != -1:
+    print("THis is not a folder. It is an .md file!")
+elif not os.path.exists(user_input):
+    print("This folder ist no folder or is not available!")
+else:
```

Comment:
You do not need this distinction, I guess. Just fill `filecut` differently, depending on the parameter the user provides in the call.